### ISSUE TITLE
Primefaces Selectonebutton gets unselect while i click on already selectedbutton!

### ISSUE THREAD

**Issue by renceabishek** [primefaces/primefaces#3048]
while i click on already selected button in primefaces selectonebutton, it gets unselect. i dont want to unselect whenever i click on selectedbutton. is there any possible solution?

**Comment by melloware** [comment]
It does not look like it unless you override the default PF javascript yourself.  Below is the code from form.selectonebutton.js that says if the button is selected if fires the unselect.

```javascript
        .on('click', function() {
            var button = $(this),
            radio = button.children(':radio');

            if(button.hasClass('ui-state-active') || radio.prop('checked')) {
                $this.unselect(button);
            }
            else {
                $this.select(button);
            }
        });
```

This is the intended behavior so adding anything that prevents it is an enhancement request.

**Comment by tandraschko** [comment]
there is a unselectable attribute, but the handling was a little bit buggy so i fixed some things for 6.2.
Next time -> questions -> forum ;)

**Comment by renceabishek** [comment]
thanks mate. i got it
